src/hooks/getFormatDate.ts

```diff
@@ -0,0 +1,7 @@
+export default function getFormatDate(value: Date) {
```

Comment:
사소하지만 get, format 둘 다 동사여서 요렇게 표현되는게 좋을 것 같아요!
```suggestion
export default function getFormattedDate(value: Date) {
```

Reply:
전혀 사소하지 않습니다 감사합니다